### ISSUE TITLE
mapping with BWA versus LONGRANGER ALIGN

### ISSUE THREAD

**Issue by burri-wildlight** [bcgsc/arcs#91]
Hi,

I would be curious to hear whether ARCS also accepts bam files produced by longranger align rather than bwa. If not, what makes the difference?

I am asking because I would expect scaffolding with mappings based on the Lariat aligner implemented in longranger that resolves ambiguous mappings be more accurate, wouldn't they?

Thanks a lot for any insights!
Reto

**Comment by burri-wildlight** [comment]
Sorry, just found my response...!

**Comment by lcoombe** [comment]
Glad you found you answer @burri-wildlight! If anyone else is wondering, I answered a similar question in #71

**Comment by burri-wildlight** [comment]
Hi all,

I have run some tests relating to the above question and to #71. Below the results, in case these can be of use to somebody. I've run parameter sweeps on the lower bound of m (5, 10, 20, 30, 50), e (30, 50, 75, 100, 125, 150, 200, 250 kb), and s (95, 98), and on bwa vs Lariat (longranger align). LINKS settings were l=5 and A=0.9. The first (m) seems to have no effect whatsoever. Results for the rest are provided below.

In a nutshell, we get more scaffolding and higher N-values at high e and with Lariat.

Some background on the choice of e: We have both a rather contiguous primary assembly (contig N50 >8 Mb) and ultra-HMW DNA, for which reason I think it is reasonable to go quite high with this parameter. Thoughts and comments welcome!

[ARCS+LINKS_parameter-sweep.pdf](https://github.com/bcgsc/arcs/files/4573216/ARCS%2BLINKS_parameter-sweep.pdf)

**Comment by lcoombe** [comment]
Thanks for sharing your results, @burri-wildlight!
